amt_test.go

```diff
@@ -53,14 +54,17 @@ func runBenchmarkWithBitWidths(b *testing.B, bitwidths []uint, fn func(*testing.
 
 type mockBlocks struct {
 	data               map[cid.Cid]block.Block
+	dataMu             sync.Mutex
```

Comment:
required to prevent race condition in test since `ParallelDiff` accesses the store in parallel